[2024-12-17T15:47:35.579+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_data_pipeline.validate_data manual__2024-12-17T15:43:45+00:00 [queued]>
[2024-12-17T15:47:35.591+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_data_pipeline.validate_data manual__2024-12-17T15:43:45+00:00 [queued]>
[2024-12-17T15:47:35.592+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-12-17T15:47:35.609+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): validate_data> on 2024-12-17 15:43:45+00:00
[2024-12-17T15:47:35.614+0000] {standard_task_runner.py:57} INFO - Started process 257 to run task
[2024-12-17T15:47:35.619+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'covid_data_pipeline', 'validate_data', 'manual__2024-12-17T15:43:45+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/covid_pipeline_dag.py', '--cfg-path', '/tmp/tmp9gx7w84c']
[2024-12-17T15:47:35.623+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask validate_data
[2024-12-17T15:47:35.707+0000] {task_command.py:416} INFO - Running <TaskInstance: covid_data_pipeline.validate_data manual__2024-12-17T15:43:45+00:00 [running]> on host bee46ca94d16
[2024-12-17T15:47:35.830+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ctw2012105@@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_data_pipeline' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T15:43:45+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-17T15:43:45+00:00'
[2024-12-17T15:47:35.864+0000] {data_validator.py:34} INFO - Validating data schema
[2024-12-17T15:47:35.865+0000] {data_validator.py:50} INFO - Validating data quality
[2024-12-17T15:47:35.869+0000] {data_validator.py:82} INFO - All validations passed successfully
[2024-12-17T15:47:35.870+0000] {python.py:194} INFO - Done. Returned value was: Data validation completed
[2024-12-17T15:47:35.906+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=covid_data_pipeline, task_id=validate_data, execution_date=20241217T154345, start_date=20241217T154735, end_date=20241217T154735
[2024-12-17T15:47:35.962+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-17T15:47:35.991+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
